.gitignore

```diff
@@ -29,3 +29,6 @@ src/main/resources/
 src/main/secrets/
 
 .idea
+
+# Prevent accidental edits to run script
```

Comment:
I think if it's in the repo, it ought to be tracked by GIT, so I'd rm this

Reply:
My reason behind putting it in the gitignore is that the domain name for the dev server cloudshell uses changes pretty frequently, but I can remove this

Reply:
So are you changing this script frequently to account for that? If so, I would either try to make the script more generic so that it doesn't have to be updated to support the changes in cloudshell dev server, or don't include it in the repo.

Reply:
I'm going to mark this as a draft for now because it's not too essential but I get what you're saying